.github/workflows/scheduled.yml

```
@@ -396,3 +403,46 @@ jobs:
           name: join-fuzzer-failure-artifacts
           path: |
             /tmp/join_fuzzer_repro
+
+  linux-exchange-fuzzer-run:
+    runs-on: ubuntu-latest
+    needs: compile
+    timeout-minutes: 120
+    steps:
+
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+        with:
+          ref: "${{ inputs.ref || 'main' }}"
```

```suggestion
          ref: "${{ inputs.ref  }}"
```
Otherwise this will not checkout the PR ref